test/unit_test/deploy/test_migration_volumes.py

```
@@ -0,0 +1,182 @@
+MIGRATION_VOLUMES_SH = REPO_ROOT / "docker" / "migration_volumes.sh"
+MIGRATION_SH = REPO_ROOT / "docker" / "migration.sh"
+
+
+def _call_get_index_volume(engine: str) -> str:
+    """Source ``migration_volumes.sh`` in a sub-bash and call
+    ``get_index_volume_for_engine`` for the given DOC_ENGINE.
+    Returns the function's stdout (with trailing whitespace stripped).
+    """
+    if not MIGRATION_VOLUMES_SH.is_file():
+        pytest.fail(f"missing helper: {MIGRATION_VOLUMES_SH}")
+    out = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)